src/plugins/kibana/public/dashboard/services/_saved_dashboard.js

```diff
@@ -57,7 +57,7 @@ module.factory('SavedDashboard', function (courier, config) {
     timeRestore: 'boolean',
     timeTo: 'string',
     timeFrom: 'string',
-    refreshInterval: 'string'
+    refreshInterval: 'object'
```

Comment:
This might be one way to do it: https://github.com/elastic/kibana/blob/master/src/ui/public/courier/saved_object/saved_object.js#L76

Not sure if that's a valid syntax for the expandShorthand method or not.